lib/astarte/client/appengine/devices.ex

```diff
@@ -106,6 +176,36 @@ defmodule Astarte.Client.AppEngine.Devices do
     end
   end
 
+  @doc """
+  Returns values for a given datastream interface.
+
+  ## Options
+
+    * `:path` - endpoint path
+
+    * `:query` - list of query params
+
+  ## Query options
```

Comment:
In all the examples, don't use the atom when using it as value, i.e. remove the `:` in front.
For example:
query all values since a certain timestamp (all entries where timestamp >= `:since`
becomes
query all values since a certain timestamp (all entries where timestamp >= `since`

And so on for all the others